api/tests/test_rag_evaluation.py

```
@@ -0,0 +1,309 @@
+    """Test version detection accuracy evaluation."""
+
+    @pytest.mark.asyncio
+    async def test_version_detection_requires_detector(self, mock_rag_service):
+        evaluator = RAGEvaluator(mock_rag_service)
+        with pytest.raises(ValueError, match="Version detector not provided"):
+            await evaluator.run_version_detection_tests(TEST_QUESTIONS)
+
+    @pytest.mark.asyncio
+    async def test_version_detection_accuracy(self, evaluator, mock_version_detector):
+        # Setup detector to return correct versions (version, confidence, clarifying_question)
+        async def mock_detect(question, history):
```

Ruff (0.14.10)

Unused function argument: `history`

(ARG001)